sdk/src/main/java/software/amazon/awssdk/iot/AwsIotMqttConnectionBuilder.java

```diff
@@ -176,6 +188,58 @@ public static AwsIotMqttConnectionBuilder newMtlsWindowsCertStorePathBuilder(Str
         }
     }
 
+    /**
+     * Create a new builder with mTLS, using a certificate and key stored in a Java keystore.
+     *
+     * @param keystorePath Path to the Java keystore on the file system.
+     * @param keystorePassword The password for the Java keystore.
+     * @param keystoreFormat The format of the Java keystore. Set to 'default' to use the default Java keystore type.
+     * @param certificateAlias The alias of the certificate and key to use with the builder.
+     * @param certificatePassword The password of the certificate and key to use with the builder.
+     * @return {@link AwsIotMqttConnectionBuilder}
+     */
+    public static AwsIotMqttConnectionBuilder newJavaKeystoreBuilder(String keystorePath, String keystorePassword, String keystoreFormat, String certificateAlias, String certificatePassword) {
```

Comment:
why not just accept the `KeyStore` directly as an input? That seems much more reasonable.

Reply:
Hmm, good point. Changed it to use a `KeyStore` as the input and added a comment that it needs to be loaded. 👍